src/app/views/trip_list_view.js

```diff
@@ -4,18 +4,25 @@ import TripView from 'app/views/trip_view';
 
 const TripListView = Backbone.View.extend({
   initialize: function(){
+
     this.detailsTemplate = _.template(Backbone.$('#tmpl-trip-details').html());
-    // this.detailsModal = this.$('#trip-details');
-    // this.detailsModal.hide();
+    this.detailsModal = this.$('#trip-details');
+    this.detailsModal.hide(); // Modal starts hidden
+
+    this.listenTo(this.model, 'update', this.render);
+  },
+
+  events: {
+    'click .trip-card': 'showDetails'
```

Comment:
I'm not sure why we need this event, it seems like the `TripView` is already listening for itself to be clicked and is triggering the `select` custom event, which we're also listening for in `TripListView`.

I'm not sure if the `return false;` in the `onClick` handler would prevent this event from triggering as well, but if it doesn't then we'll probably run `showDetails()` twice.